CMakeLists.txt

```diff
@@ -110,152 +110,42 @@ set(AZURE_STORAGE_LITE_SOURCE
   src/blob/blob_client_wrapper.cpp
 )
 
-if(APPLE)
-  option(USE_OPENSSL "Use OpenSSL instead of GnuTLS" ON)
-  add_definitions(-DUSE_OPENSSL)
-  find_package(CURL REQUIRED)
-  # Adding default brew install DIR for openssl
-  set(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR} "/usr/local/opt/openssl/")
```

Comment:
>set(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR} "/usr/local/opt/openssl/") [](start = 2, length = 67)

As talked offline, please make sure that find_openssl can find openssl installed by brew.

Reply:
I added `/usr/local/opt/openssl/` to default search path. Now it works